imgproc_test.go

```diff
@@ -1385,6 +1385,31 @@ func TestGetPerspectiveTransform(t *testing.T) {
 	}
 }
 
+func TestGetPerspectiveTransform2f(t *testing.T) {
+	src := []Point2f{
+		{0, 0},
+		{10, 5},
```

Comment:
Should these be various floating point values here so as to smoke test with more "realistic" data?

Reply:
I've updated the test, although they definitely are interpreted as `float32`s in the struct literal, even if they don't have a decimal point